src/components/NewApiDelegationPage/NewApiDelegationPage.tsx

```diff
@@ -66,8 +95,20 @@ export const NewApiDelegationsPage = () => {
             <div className={classes.pageContent}>
               <h2>Gi tilgang til API</h2>
               <h3>Velg hvilke API du vil gi tilgang til ved å klikke på pluss-tegnet.</h3>
-              <div className={classes.searchField}>
-                <SearchField></SearchField>
+              <div className={classes.searchSection}>
+                <SearchField
+                  value={searchString}
+                  onChange={(e: any) => handleSearch(e.target.value)}
+                ></SearchField>
+                <div className={classes.filter}>
+                  <Select
```

Comment:
The functionality for opening the filter alternatives when clicking any place on the Select doesn't work for me. It only works when you click on the arrow down button.

Reply:
I think that's just how the component works.
It's the same for me when I test it in the DS Storybook (https://designsystem.altinn.studio/?path=/story/components-select-multiple--multiple)